source/_integrations/oralb.markdown

```diff
@@ -48,3 +48,7 @@ The integration can discover most Bluetooth-enabled Oral-B toothbrushes. Brushes
 - Time - total brushing time in seconds.
 - Toothbrush state - whether the toothbrush is running, idle.
 - Battery - toothbrush battery percentage.
+
+<div class='note'>
+Updates of the battery sensor require an active Bluetooth connection and relatively close proximity. If you use a [bluetooth proxy](https://www.home-assistant.io/integrations/bluetooth/#remote-adapters-bluetooth-proxies), please ensure it supports active connections. All the other sensors update with active or passive connections. 
```

Comment:
This link does look like it formatted correctly in the preview

Reply:
Right. Thank you for the catch. Let me fix that.